### PR TITLE
Add com.github.alexhuntley.Plots

### PR DESCRIPTION
**Pull request by alexhuntley**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions.
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project: [upstream repo](https://github.com/alexhuntley/Plots/).
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. 

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by alexhuntley** [comment]
bot, build com.github.alexhuntley.Plots

**Comment by nedrichards** [comment]
Screenshot from 2021-01-09 20-55-42
I only get about a quarter of the graph window shown. I'm on a hidpi display if that makes any difference.

**Comment by alexhuntley** [comment]
Oops! Latest version should work better.

bot, build

**Comment by alexhuntley** [comment]
bot, build com.github.alexhuntley.Plots

File: com.github.alexhuntley.Plots.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - type: archive
+        url: http://www.gust.org.pl/projects/e-foundry/lm-math/download/latinmodern-math-1959.zip
+        sha256: aaaa060b4ffc091461e875efb9498b9abfa7c7a48f38eb33882868839903a4f8
+        dest: latinmodern-math
+cleanup:
+  - '/lib/debug'
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
```
Comment (bilelmoussaoui) [inline com.github.alexhuntley.Plots.yaml:31]:
```suggestion
  - --socket=fallback-x11
```

File: com.github.alexhuntley.Plots.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        commit: e6312e9ef274a75962de0e7b84108477d7dbdac7
+      - type: archive
+        url: http://www.gust.org.pl/projects/e-foundry/lm-math/download/latinmodern-math-1959.zip
+        sha256: aaaa060b4ffc091461e875efb9498b9abfa7c7a48f38eb33882868839903a4f8
+        dest: latinmodern-math
+cleanup:
+  - '/lib/debug'
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
```
Comment (bilelmoussaoui) [inline com.github.alexhuntley.Plots.yaml:30]:
```suggestion
```
you don't need this one if you use fallback-x11

**Comment by bilelmoussaoui** [comment]
Pretty neat app! I was wondering how can I do something like x² ? :)

**Comment by alexhuntley** [comment]
Thanks! Just type in x^2 for that :)

bot, build com.github.alexhuntley.Plots

**Comment by bilelmoussaoui** [comment]
> Thanks! Just type in x^2 for that :)
> 

It doesn't allow me to type ^ nor paste something that contains it :)

**Comment by alexhuntley** [comment]
> It doesn't allow me to type ^ nor paste something that contains it :)

Typing ^ should instead create an empty little superscript box which you can type 2 into immediately afterwards :)

**Comment by nedrichards** [comment]
cool, can confirm that hdpi issue is resolved in the latest build. this lgtm

**Comment by bilelmoussaoui** [comment]
> > It doesn't allow me to type ^ nor paste something that contains it :)
> 
> Typing ^ should instead create an empty little superscript box which you can type 2 into immediately afterwards :)

That still doesn't work, but worth an issue and not really a blocker :) 

/merge

**Comment by nedrichards** [comment]
@bilelmoussaoui the ^ thing works for me fwiw so certainly worth an upstream bug ;-)

**Comment by bilelmoussaoui** [comment]
@nedrichards probably locale related I guess, opened an upstream issue :)

**Comment by alexhuntley** [comment]
Thanks! Will look into it :)

**Pull request closed without merging**